packages/account/src/Duplicated/Utils/Validator/declarative-validation-rules.js

```diff
@@ -109,7 +109,10 @@ const initPreBuildDVRs = () => ({
         func: validAddress,
         message: localize(
             'Only letters, numbers, space, and these special characters are allowed: {{permitted_characters}}',
-            { permitted_characters: "- . ' # ; : ( ) , @ /" }
+            {
+                permitted_characters: "- . ' # ; : ( ) , @ /",
```

Comment:
(throughout) Can we do the same variable extraction as below?

Reply:
I think no because here we are inside a big object and making that a separate variable needs to put that in a far distance and makes the code less readable.